go.mod

```diff
@@ -1,46 +1,52 @@
 module github.com/konveyor/mig-controller
 
-go 1.18
+go 1.21
+
+toolchain go1.21.10
 
 require (
-	cloud.google.com/go/storage v1.10.0
+	cloud.google.com/go/storage v1.40.0
 	github.com/Azure/azure-sdk-for-go v61.4.0+incompatible
 	github.com/Azure/go-autorest/autorest v0.11.24
 	github.com/Azure/go-autorest/autorest/adal v0.9.18
 	github.com/aws/aws-sdk-go v1.35.23
 	github.com/containers/image/v5 v5.17.0
 	github.com/deckarep/golang-set v1.7.1
-	github.com/gin-contrib/cors v1.3.1
+	github.com/gin-contrib/cors v1.7.2
 	github.com/gin-gonic/gin v1.9.1
-	github.com/go-logr/logr v0.4.0
-	github.com/go-logr/zapr v0.4.0
-	github.com/google/uuid v1.3.0
+	github.com/go-logr/logr v1.4.1
+	github.com/go-logr/zapr v1.3.0
+	github.com/google/uuid v1.6.0
 	github.com/joho/godotenv v1.3.0
-	github.com/konveyor/controller v0.4.1
+	github.com/konveyor/controller v0.12.0
 	github.com/konveyor/crane-lib v0.0.11
 	github.com/konveyor/openshift-velero-plugin v0.0.0-20210729141849-876132e34f3d
-	github.com/mattn/go-sqlite3 v1.14.4
+	github.com/mattn/go-sqlite3 v1.14.22
 	github.com/onsi/ginkgo v1.16.4
-	github.com/onsi/gomega v1.16.0
+	github.com/onsi/gomega v1.33.1
 	github.com/openshift/api v0.0.0-20210625082935-ad54d363d274
 	github.com/openshift/library-go v0.0.0-20200521120150-e4959e210d3a
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.11.1
+	github.com/prometheus/client_golang v1.19.0
 	github.com/uber/jaeger-client-go v2.25.0+incompatible
 	github.com/vmware-tanzu/velero v1.7.1
-	go.uber.org/zap v1.19.0
-	golang.org/x/net v0.17.0
-	google.golang.org/api v0.56.0
-	k8s.io/api v0.22.14
-	k8s.io/apimachinery v0.22.14
-	k8s.io/client-go v0.22.14
-	k8s.io/utils v0.0.0-20211116205334-6203023598ed
-	sigs.k8s.io/controller-runtime v0.10.2
+	go.uber.org/zap v1.27.0
+	golang.org/x/net v0.26.0
+	google.golang.org/api v0.180.0
+	k8s.io/api v0.30.0
```

Comment:
These require go 1.22, so we replace them below

Reply:
That is why I stopped at 1.29, didn't know if we wanted to go to 1.22.

Reply:
For now lets stay on 1.21 to keep life easy DS